bench/topics.exs

```diff
@@ -0,0 +1,14 @@
+plain_topics = %{"topics" => T.Calls.Topics.list_topics("en")}
+fragment_topics = %{"topics" => T.Calls.Topics.topics_json_fragment("en")}
+
+Benchee.run(
+  %{
+    "encode plain topics" => fn ->
+      Jason.encode_to_iodata!(plain_topics)
+    end,
+    "encode fragment topics" => fn ->
+      Jason.encode_to_iodata!(fragment_topics)
+    end
+  },
```

Comment:
@ruslandoga  so :ets ~ "plain topics"?

Reply:
and it's the same since the most expensive operation is the benchmark appears to be encoding the list to json, and the reason `encode fragment topics` is so fast is because it returns a prerendered list of topics which can be dumped to the client on join, thus costing nearly zero both in cpu and memory

Reply:
if you rerun the benchmark without json encoding, you'd be able to see the difference between ets and plain topics